### ISSUE TITLE
Fix typo in line 239

### ISSUE THREAD

**Issue by wkumler** [wkumler/MesoscopeMetabolomicsManuscript#14]
Reviewer 2 | 04 Oct 2024 | 12:20
Line 239 Typo r = 0.697

**Comment by wkumler** [comment]
I'm actually not sure what the typo is here - line 239 currently says 

```
highly correlated with the second PC of the dataset (r = .697, % variance explained by PC2 = 15.4, 
```

which makes me think the reviewer would like to have the 0 in front of the decimal. Happy to do this if it's the norm but I thought that correlations were reported without it?

**Comment by wkumler** [comment]
Actually it seems like the convention is to report the 0 so I've changed this here and throughout the manuscript. Fixed with b09875fe.

**Comment by wkumler** [comment]
This change still needs to be propagated to the figures themselves!

**Comment by wkumler** [comment]
Fixed again with de4da21
